client/src/Filters/Language.jsx

```diff
@@ -4,11 +4,65 @@ class Language extends React.Component {
 
   constructor(props) {
     super(props);
+
+    this.state = {
+      isToggleOn: false
+    }
+
+    this.handleClick = this.handleClick.bind(this);
+    this.initialSetting = this.initialSetting.bind(this);
+  }
+
+  componentDidMount() {
+    this.initialSetting()
+  }
+
+  initialSetting() {
+    if (this.props.language === 'English') {
+      this.handleClick();
+    }
+  }
+
+  handleClick(event) {
+
+    if (event) {
+      event.preventDefault();
+    }
+
+
+
+    var setToggle = () => {
+        return new Promise ((resolve, reject) => {
+          this.setState(state => ({
+            isToggleOn: !state.isToggleOn
+          }))
+          resolve(this.state.isToggleOn)
+       })
+    }
+
+    setToggle()
+      .then ((toggle) => {
+        this.props.handleChange(this.props.season, this.state.isToggleOn, "seasons");
+      })
+      .catch( (err) => {
+        console.log(err)
+      })
+
+
   }
 
   render() {
     return (
       <div>
+      {
+        !this.state.isToggleOn &&
```

Comment:
I see you using the && operator a lot! I've never used it before! is it like an if statement kind of short circuit?